apollo-router/src/plugins/telemetry/tracing/apollo_telemetry.rs

```diff
@@ -748,7 +769,7 @@ impl Exporter {
             _ if span.attributes.contains_key(&APOLLO_PRIVATE_REQUEST) => {
                 if !self.use_legacy_request_span {
                     child_nodes.push(TreeData::Router {
-                        http: Box::new(extract_http_data(span)),
+                        http: Box::new(extract_http_data(span).0),
```

Comment:
If I cared more about the legacy code path I'd ask that we use semantic values rather than indexes here but I'm hoping we rip this code out soon anyway.

Reply:
Same